### PR TITLE
Feature testnet trading

### PR DESCRIPTION
**Pull request by walkjivefly**
This set of changes addresses https://github.com/blocknetdx/exrproxy-env/issues/85 and adds the ability to generate a testnet trading node.

### CODE REVIEW

**Review by ConanMishler — COMMENTED** [review]
Wow, you did a lot of detailed work here! All the changes look good to me just reviewing them here, but I really should test them to make sure you didn't seriously break anything. I'll try to test & approve tomorrow.

**Comment by ConanMishler** [comment]
I've tested this PR (#87) and I can confirm that it doesn't break anything in the existing snode deployment workflow. (I didn't test that it adds the new `testTNODE` feature as expected, but I'm sure @walkjivefly has tested that.) I would approve this PR and merge it except I know @pom11 has at least one PR waiting to be approved (#89), which will have conflicts with this PR. @pom11 's PR #89 looks pretty clean/good to me, so I'm guessing it will be approved any day now? If #89 will indeed be approved/merged very soon, then I feel it will be better to wait for #89 to be merged, then let @walkjivefly pull the latest `master`, resolve the conflicts locally, then include his conflict-resolution commit into this PR.

I'm not sure how PR #70 from @pom11 relates to this PR (#87). #70 looks like it is pretty far out of sync with base/master, so I'm wondering if @atcsecure will close it? Or if @pom11 will pull the latest `master` to the `dev-xquery-pom` branch and thereby resync `dev-xquery-pom` with `master` ?

File: autobuild/app.py
Comment (ConanMishler) [inline autobuild/app.py:84]: I know there aren't many comments in this code, but it might be nice to add a comment here (lines 79-84), giving an example of a non-standard blockchain name which this enhancement will allow us to accommodate.
I guess the new `daemon_stem` field of `manifest-latest.json` should also be documented in the README of the `blockchain-configuration-files` branch, no?

**Review by ConanMishler — CHANGES_REQUESTED** [review]

File: autobuild/app.py
Comment (walkjivefly) [reply autobuild/app.py]: Added minimal doc.

Yeah, there'll be a couple more PRs forthcoming for the related `blockchain-configuration-files` and `dockerimages` changes

**Review by ConanMishler — APPROVED** [review]

**Pull request merged**
